README.md

```diff
@@ -66,6 +66,8 @@ Unit tests need to run in `sudo`, as they rely on eBPF: `sudo go test ebpf-detec
     - GKE (Ubuntu and Alpine Linux, AMD64)
     - EKS (Ubuntu and Alpine Linux, AMD64)
     - Ubuntu 22.04 (ARM64)
+* Tried to test in the following environments:
```

Comment:
I'd prefer listing this under the prerequisite of support for eBPF by the linux distro.